src/index.js

```diff
@@ -240,16 +240,20 @@ rtm.on(RTM_EVENTS.MESSAGE, (message) => {
     }
   } else if (message.text.toLowerCase() === 'help') {
     rtm.sendMessage('*command list*\n' +
-        '> feel [emoji] [1-6] ([snippet]) `tell the bot how you feel now, snippet is optional`\n' +
+        '> feel/felt [emoji] [1-6] ([snippet]) `tell the bot how you feel/felt now, snippet is optional`\n' +
         '> echo `get the average mood from the past week` \n' +
         '> history `get the mood from the past week` \n' +
         '> quotes `get the snippets from the past week` \n' +
         '> hello `say hi to me and i will tell you how i feel` \n' +
         '> help `get help info`\n' +
         '```1 (depressed), 2 (sad), 3 (unhappy), 4 (satisfied), 5 (joyful), 6 (exuberant)```'
       , message.channel);
-  } else if (message.text.substr(0, 4).toLowerCase() === 'feel') {
-    const timestamp = getTimestampInSeconds(message);
+  } else if (message.text.substr(0, 4).toLowerCase() === 'feel' || message.text.substr(0, 4).toLowerCase() === 'felt') {
+    const command = message.text.substr(0, 4).toLowerCase();
+    let timestamp = getTimestampInSeconds(message);
+    if (command === 'felt') {
+      timestamp -= 86400;
+    }
```

Comment:
This is a trick to handle two separate commands in the same else if branch.
What about treating `feel` and `felt` as separate commands? 
This would simplify a bit the implementation of each one. We could then extract the common parts in functions.

Reply:
I tried to reuse the code, because these two commands are more-or-less the same